subsys/bluetooth/mesh/settings.c

```diff
@@ -61,7 +61,7 @@ static int mesh_commit(void)
 	}
 
 	if (IS_ENABLED(CONFIG_BT_MESH_PB_GATT)) {
-		bt_mesh_proxy_prov_disable(true);
+		(void)bt_mesh_proxy_prov_disable();
```

Comment:
Should also disconnect?

Reply:
In this mesh_commit, are not all calls after power up? Will there be a connection at this time?